.travis.yml

```diff
@@ -7,6 +10,19 @@ cache:
 before_install:
   - true && `base64 --decode <<< ZXhwb3J0IFNBVUNFX1VTRVJOQU1FPUltcHJvYmFibGVFbmdCb3Q=`
   - true && `base64 --decode <<< ZXhwb3J0IFNBVUNFX0FDQ0VTU19LRVk9M2RmOGVkMGMtMThmNS00YWU4LTg4NTItZWMxMjZkZDYwMjQz`
-install: npm install
+install:
+  - go get github.com/Sirupsen/logrus
+  - go get github.com/grpc-ecosystem/go-grpc-prometheus
+  - go get github.com/mwitkow/go-conntrack
+  - go get github.com/mwitkow/go-conntrack/connhelpers
+  - go get github.com/mwitkow/go-grpc-middleware
+  - go get github.com/mwitkow/go-grpc-middleware/logging/logrus
+  - go get github.com/mwitkow/grpc-proxy/proxy
+  - go get github.com/prometheus/client_golang/prometheus/promhttp
+  - go get github.com/spf13/pflag
+  - go get github.com/rs/cors
+  - go get github.com/golang/protobuf/protoc-gen-go
+  - go get github.com/stretchr/testify
+  - npm install
```

Comment:
Assumed that my usage of `go_import_path` probably means that I have to use `language: go`